MODULE.bazel

```diff
@@ -8,6 +8,10 @@ bazel_dep(name = "bazel_skylib", version = "1.4.2")
 bazel_dep(name = "platforms", version = "0.0.6")
 
 cuda = use_extension("@rules_cuda//cuda:extensions.bzl", "toolchain")
+cuda.local_toolchain(
+    name = "local_cuda",
+    toolkit_path = "",
```

Comment:
If a user doesn't have an install at the default path does this cause a failure if they also override it in their own module file? Should we be loading the extension with dev_dependency = True to avoid this if so?

Reply:
No, this does not trigger analysis error anymore thanks to #209. The generated `local_cuda` repo is as follows
```
root@c578fa5b9b4e ~/rules_cuda (cloudhan/our-module-local-cuda)# tree bazel-rules_cuda/external/_main\~toolchain\~local_cuda/
bazel-rules_cuda/external/_main~toolchain~local_cuda/
|-- BUILD -> /root/rules_cuda/cuda/runtime/BUILD.local_cuda_disabled
|-- REPO.bazel
|-- WORKSPACE
|-- defs.bzl
`-- toolchain
    |-- BUILD
    |-- clang
    |   `-- BUILD
    `-- disabled
        `-- BUILD -> /root/rules_cuda/cuda/templates/BUILD.local_toolchain_disabled

3 directories, 7 files
root@c578fa5b9b4e ~/rules_cuda (cloudhan/our-module-local-cuda)# cat bazel-rules_cuda/external/_main\~toolchain\~local_cuda/BUILD
load("@bazel_skylib//rules:common_settings.bzl", "bool_setting")

package(
    default_visibility = ["//visibility:public"],
)

bool_setting(
    name = "valid_toolchain_found",
    build_setting_default = False,
)
````